web-portal/pages/subscribe.vue

```diff
@@ -26,7 +26,7 @@
 
 <script>
   export default {
-    layout: 'no-subscribe'
+
```

Comment:
do we still need this script if it's empty?

Reply:
Probably not.